content/guides/models/track/runs/multiple-runs-per-process

```diff
@@ -0,0 +1,147 @@
+# Multiple Active Runs in One Process (Reinit Behavior)
+
+**Quick Reference**  
+- **`return_previous`**: Return the existing run; no new run is created.  
+- **`finish_previous`**: Finish (end) the old run, then create a new one.  
+- **`create_new`**: Create a new run without finishing the old one—great for parallel or interleaved runs, but watch out for integration compatibility.
+
+---
+
+By default, W&B assumes each Python process has only **one active run** at a time. Calling `wandb.init()` again, in the default scenario, will either return the same run or finish the old run before starting a new one—depending on the configuration. However, there are workflows where you might want **multiple runs** in the same Python process:
+
+- Keeping a single “primary” run active throughout a script while spinning up short-lived “secondary” runs for evaluations or sub-tasks.  
+- Orchestrating sub-experiments in a single file.  
+- Logging from one “master” process to several runs that represent different tasks or time periods.
+
+W&B provides a **`reinit`** setting to control how `wandb.init()` behaves when there’s an existing active run. Below, we detail each `reinit` option and show how to maintain multiple concurrent runs—finishing them at different times without interference.
+
+---
+
+## Overview of `reinit` Options
+
+Use the `reinit` argument when calling `wandb.init()`, or set it via environment variable (e.g., `WANDB__INIT__REINIT`). The valid options:
+
+1. **`return_previous`**  
+   - If another run is active, return *that same* run.  
+   - *No new run is created.* This is effectively a “no-op” and is the legacy default.
+
+2. **`finish_previous`**  
+   - *Finish* (calls `wandb.finish()`) on any active run(s) before creating a new one.  
+   - Handy if you want a clear break between sequential sub-runs.
```

Comment:
Don't mention `wandb.finish()`, we are trying to deprecate global run functions.

```suggestion
   - *Finish* (calls `run.finish()`) on any active run(s) before creating a new one.  
   - Handy if you want a clear break between sequential sub-runs.
   - This is the default when running in a Jupyter/IPython notebook.
```